src/libraries/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -42,7 +40,6 @@ private enum CookieVariant
 
         private static Func<Cookie, CookieVariant> s_getVariantFunc;
 
-        [DynamicDependency("get_Variant", typeof(Cookie))]
```

Comment:
I pinged @scalablecory offline for a suggestion for how to test this change. I'll follow up with a PR when we figure one out.

Reply:
Logged issue - https://github.com/dotnet/runtime/issues/38447.